content/03.use-cases-a.md

```diff
@@ -13,34 +13,34 @@ Version control is particularly useful if a mistake has been made where a user h
+Digital lab notebooks, stored in the cloud, provide clear benefits given the ease with which documents can be shared with new employees and updated as policy changes or experimental methods are modified [@doi:10.1371/journal.pcbi.1004385].
+Increasingly, researchers are leveraging GitHub's underlying version control to keep and share digital lab notebooks [@doi:10.1038/538127a].
 At a minimum, commit statements can provide a record of daily changes made to any code stored on GitHub [@doi:10.1186/1751-0473-8-7].  
-GitHub issues can be used to track and prioritize lab objectives and goals, as well as tracking any status updates.  
-Some EEB labs have even turned their lab notebooks into shareable websites [@url:https://scheuerell-lab.github.io/lab-book/; @url:https://github.com/HuckleyLab/how_we_work] as a centralized location for all lab resources.  
+GitHub issues can be used to track and prioritize lab objectives and goals, as well as tracking any status updates.
+Some EEB labs have even turned their lab notebooks into shareable websites [@url:https://scheuerell-lab.github.io/lab-book; @url:https://github.com/HuckleyLab/how_we_work] as a centralized location for all lab resources.  
 
 ### Classroom teaching / educational materials
 <!-- *contributors to this section: Cole Brookson* -->
 
-GitHub provides a large variety of uses for hosting teaching/educational materials. 
-In fact, through taking advantage of the suite of GitHub features, the entire process of running a course, workshop, or even just a lecture, can all be done openly on GitHub.  
-As a matter of gross simplification, organizing a course (for example) could be broken down into: 1) developing the material (i.e., slides, examples, relevant readings, labs, etc.), 2) hosting the course on some online platform for students to access, 3) delivering the content, and 4) accepting student work submissions and then returning graded material.  
+GitHub provides a large variety of uses for hosting teaching/educational materials.
+In fact, through taking advantage of the suite of GitHub features, the entire process of running a course, workshop, or even just a lecture, can all be done openly on GitHub.
+As a matter of gross simplification, organizing a course (for example) could be broken down into: 1) developing the material (i.e., slides, examples, relevant readings, labs, etc.), 2) hosting the course on some online platform for students to access, 3) delivering the content, and 4) accepting student work submissions and then returning graded material.
 While of course there are other purpose-built platforms for this type of activity, few of them provide the usability at the price point GitHub does. 
 
-First, developing course material, from slides to labs and everything in between, can be done on GitHub, out in the open, where others can see, review and offer feedback on your process!  
```

Comment:
@BrandonEdwards Before, I was thinking that the double space at the end of each line was necessary for github to recognize a line break. But looks like that's not the case! Thanks for making the changes.